testdata/config.toml

```diff
@@ -10,4 +10,14 @@ ipapico_enable = false
 ipapicom_enable = false
 
 [ips_config]
-
+peer_file_path = "testdata/peers.json"
+pref_location = true
+pref_location_closer = true
+pref_location_distance = 1000
```

Comment:
Would be nice to have comments in this file, as the real user won't read the code in search of the documentation

Reply:
some fields were renamed (above).  `config.toml` is just not yet updated.
```
use_geolocation: true,
use_closer_geolocation: true,
geolocation_minmax_distance_km: 1000,
```

Reply:
Done.